README.md

```diff
@@ -15,108 +17,31 @@ After building your image, use [push-to-registry](https://github.com/redhat-acti
-    </td>
-  </tr>
-
-  <tr>
-    <td>context</td>
-    <td>No</td>
-    <td>Path to directory to use as the build context.<br>
-    Default: <code>.</code></td>
-  </tr>
-
-  <tr>
-    <td>build-args</td>
-    <td>No</td>
-    <td>Build arguments to pass to the Docker build using <code>--build-arg</code>, if using a Dockerfile that requires ARGs.<br>
-    Uses the form <code>arg_name=arg_value</code>, and separate arguments with newlines.</td>
-  </tr>
-
-  <tr>
-    <td>content</td>
-    <td>No</td>
-    <td>The content to copy inside the container to create the final image. This is a multiline input to allow you to copy more than one file/directory.<br>
-    <pre>content: |
-  target/spring-petclinic-2.3.0.BUILD-SNAPSHOT.jar</pre>
-    </td>
-  </tr>
-
-  <tr>
-    <td>entrypoint</td>
-    <td>No</td>
-    <td>The entry point to set for the container. This is a multiline input; split arguments across lines.
-      <pre>entrypoint: |
-  java
-  -jar
-  spring-petclinic-2.3.0.BUILD-SNAPSHOT.jar</pre>
-    </td>
-  </tr>
-
-  <tr>
-    <td>port</td>
-    <td>No</td>
-    <td>The port to expose when running the container.</td>
-  </tr>
-
-  <tr>
-    <td>workdir</td>
-    <td>No</td>
-    <td>The working directory to use within the container.</td>
-  </tr>
-
-  <tr>
-    <td>envs</td>
-    <td>No</td>
-    <td>The environment variables to be set when running the container. This is a multiline input to add multiple environment variables.<br>
-      <pre>
-envs: |
-  GOPATH=/root/buildah/go</pre>
-    </td>
-  </tr>
-</table>
+### Inputs for build from dockerfile
+
+| Input Name | Description | Default |
+| ---------- | ----------- | ------- |
+| build-args | Build arguments to pass to the Docker build using `--build-arg`, if using a Dockerfile that requires ARGs. Uses the form `arg_name=arg_value`, and separate arguments with newlines. | None
+| context | Path to directory to use as the build context. | `.`
+| dockerfiles | The list of Dockerfile paths to perform a build using docker instructions. This is a multiline input to allow multiple Dockerfiles. | **Must be provided**
+| image | Name to give to the output image. | **Must be provided**
+| oci | Build the image using the OCI format, instead of the Docker format. By default, this is `false`, because images built using the OCI format have issues when published to `Dockerhub`. | `false`
+| tags | The tags of the image to build. For multiple tags, separate by a space. For example, `latest ${{ github.sha }}` | `latest`
+
+### Inputs for build without dockerfile
+
+| Input Name | Description | Default |
+| ---------- | ----------- | ------- |
+| base-image | The base image to use for the container. | **Must be provided**
+| content | The content to copy inside the container to create the final image. This is a multiline input to allow you to copy more than one file/directory.| No
```

Comment:
"Paths to files or directories to copy inside the container to create the file image. This is a multiline input to allow you to copy multiple files/directories."